ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/shared-phz/sts/provision/cucushift-installer-rehearse-aws-ipi-shared-phz-sts-provision-chain.yaml

```diff
@@ -0,0 +1,35 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-shared-phz-sts-provision
```

Comment:
compared this chain with `cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-provision`, the only difference is setting `ENABLE_SHARED_VPC` to `no`, if that, maybe we can directly call `cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-provision`, and overriding `ENABLE_SHARED_VPC` with a default value in install workflow.

Reply:
Updated